### ISSUE TITLE
Implement UI Context Responses for Pause/Play and Step

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#317]
Following up to comment in https://github.com/phetsims/molecules-and-light/issues/299#issuecomment-604927266

Based on interviews with non-visual learners the Pause/Play and Step Forward buttons were not very intuitive to use.  This may have been in part due to the lack of context responses to communicate state changes. 

To move forward with making this sim more accessible to non-visual learners, we will implement the UI Context Responses in these 2 tables in the design doc:
- [Table: Pause/Play  Button](https://docs.google.com/document/d/1r9D3vq4QvmmXkQ5G6HexpybUv_vL6LrYFb3wgPfUdWg/edit#heading=h.mmb8tcjfoy8p)
- [Table: Step Forward Button](https://docs.google.com/document/d/1r9D3vq4QvmmXkQ5G6HexpybUv_vL6LrYFb3wgPfUdWg/edit#heading=h.qxyb5aefea91)

We will also investigate in another issue an option to disable the Pause/Play button in Time Control Node which could eliminate the need for these UI Context Responses for non-visual learners and improve the overall sim experience for everyone.

**Comment by terracoda** [comment]
@jessegreenberg, there are sim-specific and general responses in the table. 

If it is not too much work, could you implement the sim-specific responses for now? 

AND then we can review the general responses with others on the team when we discuss updates to Time Control Node?

**Comment by jessegreenberg** [comment]
Done in the above commit, assigning to @terracoda to review.

**Comment by terracoda** [comment]
This Pause/Play sounds great. I tested with both the regular toggle button and the the on/off switch. Both sound great. 

I didn't make an issue for the Step Forward button. The Step Forward button has one context response in the case where the user tries to use the enabled Step Forward button when Light source is actually off. See[Table: Step Forward Button](https://docs.google.com/document/d/1r9D3vq4QvmmXkQ5G6HexpybUv_vL6LrYFb3wgPfUdWg/edit?ts=5e8cacd8#heading=h.qxyb5aefea91) for details.

I currently do not hear the following when the Light Source is off and I try to use the Step Forward button. 
- "Turn light source on to use Step Forward."

**Comment by jessegreenberg** [comment]
Ah, thanks @terracoda, pushed above. There is now an alert for "Turn light source on to use Step Forward." but the rest haven't been added yet, those will come when the other alerts have been implemented.

**Comment by jessegreenberg** [comment]
Also assigning to @Matthew-Moore240 in case he has time to review first.

**Comment by jessegreenberg** [comment]
Reassigning to myself, Ill reassign to review once the step alerts are done.

**Comment by jessegreenberg** [comment]
These alerts are complete, and can be reviewed at
https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.46/phet/molecules-and-light_a11y_view.html

**Comment by terracoda** [comment]
@jessegreenberg, the UI context responses that support the use of the Pause/Play and Step Forward buttons sounds great to me.

Marking as resolved.
